content/en/monitors/types/process.md

```diff
@@ -19,22 +19,43 @@ further_reading:
 
 ## Overview
 
-Live process monitors are based on data collected by the [Process Agent][1]. Create monitors that warn or alert based on the count of any group of processes across hosts or tags.
+Live Process Monitors are based on data collected by the [Process Agent][1]. Create monitors that warn or alert based on the count of any group of processes across hosts or tags.
+
+Live Process Monitors are recommended for the following use cases:
+
+- Ensure that enough instances of a non-containerized process are running.
+- Flag when a specific process is executed.
```

Comment:
```suggestion
- Flag when a specific process is running.
```

Reply:
I'm just curious, why "executed" over "running"?

Reply:
I don't remember what the exact reason was, but I believe `running` is more accurate since lp monitors don't alert on process events.

Reply:
That's correct, we don't detect process execution currently, we only collect processes that are running for two checks (typically 20 seconds).

I asked because the suggestion was not committed but resolved, but it looks like you accepted the suggestion in the meantime